chart/README.md

```diff
@@ -366,6 +366,17 @@ helm install airflow . \
     --set data.brokerUrl=redis://redis-user:password@redis-host:6379/0
 ```
 
+## Using additional containers
+
+If you are using your own sidecar container, you can add it through the `extraContainers` value. You can define different containers for scheduler, webserver and worker pods. For example, a sidecar that syncs DAGs from object storage.
+
+```bash
```

Comment:
@Swalloow  How to distinguish for example which container is for the sheduler and which one is for the webserver?

Reply:
@gardnerdev `extraContainers` are defined as properties of scheduler, webserver, and worker.

```
scheduler:
  ...
  extraContainers: []

webserver:
  ...
  extraContainers: []

workers:
  ...
  extraContainers: []
```

Reply:
Thanks, looks awesome

Reply:
@Swalloow could you put in docs or here how you implemented sindecar container pattern? I mean mounting volumes example

Reply:
sidecar container pattern can be implemented in many ways.
I'm not sure if it is correct to write a specific example here.

Reply:
Concrete examples are very helpful, especially since file syncing is a very common case in Airflow.  If we use specific examples, we should check if they really work.